safepickle/__init__.py

```diff
@@ -1,8 +1,9 @@
 from .safepickle import load, loads, dump, dumps
+from .safepickle import PickleError, PicklingError, UnpicklingError
 
 
 __title__ = 'safepickle'
-__version__ = '0.1.0rc2'
+__version__ = '0.1.0rc3'
```

Comment:
Don't version bump in the PRs, we will let Jenkins handle that when the release happens